media/css/pocket/components/about.scss

```diff
@@ -182,7 +182,6 @@ $pocket-image-path: '/media/img/pocket';
             height: 590px;
             right: -155px;
             width: 630px;
-            z-index: -1;
```

Comment:
@nathan-barrett can you remember why the negative z-index was needed? I'm still puzzled over why it was preventing the images from appearing in this branch.

Reply:
Just looked, and as far as I can tell, I added this rule so that the image from the next section could overlap. Looking now, its not a great pattern and easily something that I could've borrowed from the original styles